crates/subspace-networking/src/bin/subspace-bootstrap-node/main.rs

```diff
@@ -10,18 +10,30 @@ use libp2p::metrics::Metrics;
 use libp2p::{identity, Multiaddr, PeerId};
 use prometheus_client::registry::Registry;
 use serde::{Deserialize, Serialize};
+use std::collections::HashSet;
 use std::error::Error;
 use std::fmt::{Display, Formatter};
 use std::net::SocketAddr;
+use std::num::NonZeroUsize;
 use std::sync::Arc;
+use std::time::Duration;
 use subspace_metrics::{start_prometheus_metrics_server, RegistryAdapter};
 use subspace_networking::libp2p::multiaddr::Protocol;
-use subspace_networking::{peer_id, Config, KademliaMode};
+use subspace_networking::utils::strip_peer_id;
+use subspace_networking::{
+    peer_id, Config, KademliaMode, NetworkingParametersManager, NetworkingParametersManagerConfig,
+};
 use tracing::{debug, info, Level};
 use tracing_subscriber::fmt::Subscriber;
 use tracing_subscriber::util::SubscriberInitExt;
 use tracing_subscriber::EnvFilter;
 
+/// Defines an expiration period for the peer marked for the removal for Kademlia DHT.
+const REMOVE_KNOWN_PEERS_GRACE_PERIOD_FOR_KADEMLIA_SECS: Duration = Duration::from_secs(3600);
+
+/// Size of the LRU cache for peers.
+pub const PEER_CACHE_SIZE: NonZeroUsize = NonZeroUsize::new(10000).expect("Not zero; qed");
```

Comment:
I think I'd rename this to `KNOWN_PEERS_CACHE_SIZE`, `NetworkingParametersManagerConfig` to `KnownPeersConfig` and `NetworkingParametersManager` to `KnownPeers` (of maybe retain `Manager` suffix for this and its config).